api/src/main/webapp/WEB-INF/cron_default.yaml

```diff
@@ -76,6 +76,13 @@ cron:
     that is (1) after the close of normal business working hours, and (2) early enough in the evening that the
     entire export (and downstream data flows) can complete before start of the next business day.
   url: /v1/cron/exportToRdr
+  schedule: every day 22:00
+  timezone: America/Chicago
+  target: api
+- description: >
+    Update Research Purpose Review for each workspace to true, if the workspace has been created for more than 17 or 365 days
```

Comment:
nit: this sounds like it automatically reviews the research purpose twice. this should say something more like:
'Marks a workspace to prompt the user to review the research purpose, if the workspace has existed for more than 15 days without research purpose review or if the research purpose has not been reviewed int he past 365 days.'